extras/AFC_stepper.py

```diff
@@ -238,10 +238,15 @@ def prep_callback(self, eventtime, state):
                 if self.load_state == True and self.prep_state == True:
                     self.status = 'Loaded'
                     self.AFC.afc_led(self.AFC.led_ready, led)
-            else:
```

Comment:
This else statement need to stay, if a lane is ejected the led stays green

Reply:
Done